src/graphorm/index.ts

```diff
@@ -177,6 +179,15 @@ const createSmartTitleField = ({ field }: { field: string }): GraphORMField => {
   };
 };
 
+const organizationLink = (type: OrganizationLinkType) => ({
+  jsonType: true,
+  select: 'links',
+  transform: (
+    links: OrganizationLink[] | null,
+  ): OrganizationLink[] | undefined =>
+    links?.filter((link) => link.type === type),
+});
```

Comment:
Alternative is a more complex select that only selects the jsonb objects with the specifc type. But since it's maybe a couple few objects, I think a quick transform with filter is fine.